remix.config.js

```diff
@@ -1,9 +1,9 @@
 const baseConfig =
-  process.env.NETLIFY || process.env.NETLIFY_LOCAL
+  process.env.NODE_ENV === "production"
```

Comment:
We'll only pull in the custom server settings if building for production, i.e. `ntl serve`, `ntl build`,`ntl deploy` or via git push.